Lombiq.Vsix.Orchard/Commands/InjectDependencyCommand.cs

```diff
@@ -60,6 +60,7 @@ public async Task InitializeUIAsync()
 
         private async Task MenuItemCallbackAsync()
         {
+            await ThreadHelper.JoinableTaskFactory.SwitchToMainThreadAsync();

```

Comment:
Erre miért van szükség? Ez negálja a `MenuItemCallback()`-ben lévő `JoinableTaskFactory` használatot. `LogWatcherSettingsUpdatedCallbackAsync`, `UpdateOpenErrorLogCommandAccessibilityAndTextAsync`, `LogFileWatcherBase.TimerCallbackAsync` és `GetExistingLogFilePathAsync` szintén.

Reply:
Itt is hasonló a helyzet. Ha törlöm a SwitchToMainThreadAsyc-et, akkor warning lesz:
![image](https://user-images.githubusercontent.com/18496537/155192747-c40c1844-df77-4bb2-9564-65c2e8052d22.png)

Reply:
Miután a `SwitchToMainThreadAsync()` megtörténik, mindegyik metódusban csak olyan kód van, ami aminek a main threaden kell futnia?

Reply:
Szerintem igen. A fenti példáknál csak pár property van beállítva, utána már minden lényeges kód külön thread-en van:
![image](https://user-images.githubusercontent.com/18496537/155219482-425da03b-5ce8-42b0-a424-899137b9a9be.png)
És itt is az invoke után az event handler már más threaden van
![image](https://user-images.githubusercontent.com/18496537/155219599-a78e76cb-6974-4e44-8658-cedb05582a1e.png)
![image](https://user-images.githubusercontent.com/18496537/155219751-6971089e-f1f9-425d-82fc-63629146cc48.png)

Reply:
OK akkor!